[docs] Edits to the multi-window mode docs requested by @miherlosev (#5406)
DevExpress/testcafe: docs/articles/documentation/reference/test-api/testcontroller/openwindow.md

# t.openWindow method

> Opens a new browser window. Returns the matching window descriptor, but can still be chained with other TestController methods.

Reviewer: ```suggestion
Opens a new browser window. Returns the matching window descriptor and can be chained with other TestController methods.
```
